master.toml

```diff
@@ -5,40 +5,46 @@
 
 [build.dist-7-8]
   GHC_VERSION="7.8.4"
-  CABAL_VERSION="1.22.4.0"
+  CABAL_VERSION="1.24.0.0"
   MAFIA_HAPPY = "true"
+  CACHE = "true"
 
 [build.dist-7-10]
   GHC_VERSION="7.10.2"
-  CABAL_VERSION="1.22.4.0"
+  CABAL_VERSION="1.24.0.0"
   MAFIA_HAPPY = "true"
   HADDOCK = "true"
   HADDOCK_S3 = "$AMBIATA_HADDOCK_MASTER"
   PUBLISH = "true"
   PUBLISH_S3 = "$AMBIATA_ARTEFACTS_MASTER"
   PUBLISH_EXECUTABLES = "mafia"
+  CACHE = "true"
 
 [build.dist-8-0]
   GHC_VERSION="8.0.1"
   CABAL_VERSION="1.24.0.0"
   MAFIA_HAPPY = "true"
+  CACHE = "true"
 
 [build.branches-7-8]
   GHC_VERSION = "7.8.4"
-  CABAL_VERSION = "1.22.4.0"
+  CABAL_VERSION = "1.24.0.0"
```

Comment:
Does that version of cabal work with GHC 7.8.4?

Reply:
I though it does. We'll see soon enough. If not, I'll remove caching for the 7.8 build.

Reply:
caching doesn't work for 7.8 anyway